### PR TITLE
chore(RELEASE-1040): remove file-updates

### PR DESCRIPTION
**Pull request by scoheb**
- migration performed in RELEASE-1283

### CODE REVIEW

File: internal-services/catalog/file-updates-pipeline.yaml
Comment (johnbieren) [inline internal-services/catalog/file-updates-pipeline.yaml:5]: I think we need to change the migrated version to this name before we remove this here

Comment (scoheb) [reply internal-services/catalog/file-updates-pipeline.yaml:5]: done here: https://github.com/konflux-ci/release-service-catalog/pull/682/files#diff-50b7aa963fdf55a7ea15a3ad71d8cd3b3b8d314d07fea08bf2acbbba9ef3bc68R28

**Review by johnbieren — APPROVED** [review]
Approving, but please merge it only when 682 is being merged

**Comment by johnbieren** [comment]
@scoheb we can merge this now, right?

**Comment by scoheb** [comment]
Don't we need my catalog PR to land in staging?

**Comment by johnbieren** [comment]
It actually is in staging since I promoted after it merged, but no, we remove from main when it goes to development and stable when it goes to production. See https://issues.redhat.com/browse/RELEASE-1211 for an explanation

**Pull request merged**
